packages/utils/package.json

```diff
@@ -20,7 +20,7 @@
     "chalk": "^4.1.2",
     "detect-port": "^1.3.0",
     "flatted": "^3.2.5",
-    "graphql": "^15.8.0",
+    "graphql": "15.8.0",
```

Comment:
Whats the problematic version?

Reply:
when executing `yarn why graphql`

This is the only version that is different, all others are on 15.8.0
```
├─ @types/graphql@npm:14.5.0
│  └─ graphql@npm:16.5.0 (via npm:*)
│
```